tests/system/providers/google/cloud/gcs/example_calendar_to_gcs.py

```diff
@@ -17,22 +17,30 @@
 # under the License.
 from __future__ import annotations
 
+import json
 import os
 from datetime import datetime
 
 from airflow import models
+from airflow.decorators import task
+from airflow.models import Connection
+from airflow.operators.bash import BashOperator
 from airflow.providers.google.cloud.operators.gcs import GCSCreateBucketOperator, GCSDeleteBucketOperator
 from airflow.providers.google.cloud.transfers.calendar_to_gcs import GoogleCalendarToGCSOperator
+from airflow.settings import Session
 from airflow.utils.trigger_rule import TriggerRule
 
-ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
-PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT")
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID", "default")
+PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT", "default")
 DAG_ID = "example_calendar_to_gcs"
 
 BUCKET_NAME = f"bucket_{DAG_ID}_{ENV_ID}"
-CALENDAR_ID = os.environ.get("CALENDAR_ID", "1234567890qwerty")
+CALENDAR_ID = os.environ.get("CALENDAR_ID", "primary")
 API_VERSION = "v3"
 
+CONNECTION_ID = "google_cloud_temp"
```

Comment:
```suggestion
CONNECTION_ID = f"connection_{DAG_ID}_{ENV_ID}"
```
It might be overcautious, but do you think we can do this?